examples/lib/stories/effects/effects.dart

```diff
@@ -67,5 +68,18 @@ void addEffectsStories(Dashbook dashbook) {
       'Color Effect',
       (_) => GameWidget(game: ColorEffectGame()),
       codeLink: baseLink('effects/color_effect.dart'),
+    )
+    ..add(
+      'Rotate Effect (v2)',
+      (_) => GameWidget(game: RotateEffect2Game()),
+      codeLink: baseLink('effects/rotate_effect2.dart'),
+      info: '''
+        The outer rim rotates at a different speed forward and reverse, and
```

Comment:
Add this string as a static const variable on the `RotateEffect2Game` class for improved organisation.

Reply:
ok